macosx/VDKQueue/VDKQueue.m

```diff
@@ -201,8 +201,6 @@ - (VDKQueuePathEntry *)    addPathToQueue:(NSString *)path notifyingAbout:(u_int
         
         return pathEntry;
     }
-    
-    return nil;
```

Comment:
This is third-party so we decided to keep it as is (note that it still uses ivars despite the fact that I removed them from the rest of the project earlier).

Reply:
OK, reverting.